strongbox-db-liquibase/src/main/resources/db/changelog/v1.0.0/v1.0.0.18__ArtifactGroup.xml

```diff
@@ -0,0 +1,20 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:o="http://www.unbroken-dome.org/schema/liquibase-orientdb"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd">
+
+    <changeSet id="v1.0.0.18" author="przemyslaw.fusik@gmail.com">
+
+        <o:createClass name="ArtifactGroup" abstract="true">
+            <o:property name="name" type="string" notNull="true"/>
+            <o:property name="artifactEntries" type="linkset" linkedType="ArtifactEntry"/>
+        </o:createClass>
+
+        <o:createClass name="ArtifactIdGroup" extends="ArtifactGroup"/>
+
+        <o:createIndex name="idx_artifact_id_group" on="ArtifactIdGroup" property="name" type="unique" />
```

Comment:
@fuss86 have you bean able to check that this constraint triggers only for `ArtifactIdGroup` subclass, and not for all `ArtifactGroup`?

Reply:
this idx does not exist anymore